muikku-core/src/main/java/fi/otavanopisto/muikku/schooldata/UserSchoolDataBridge.java

```diff
@@ -47,6 +48,8 @@
   
   /* User */
   
+  public BridgeResponse<List<OrganizationContactPerson>> listOrganizationContactPersonsByOrganization(Long organizationId);
```

Comment:
Change this to take in `String organizationIdentifier` which is Organization's SchoolDataIdentifier's identifier (string) instead. SchoolDataBridge's are expected to do the conversion from this identifier to the underlying system's format instead.